models/comments.model.js

```diff
@@ -12,28 +12,30 @@ exports.insertComment = async (comment) => {
     });
   }
 
-  if (await checkUserExists(username)) {
-    return Promise.reject({
-      status: 404,
-      msg: "User not found",
-    });
-  }
+  // if (await checkUserExists(username)) {
+  //   return Promise.reject({
+  //     status: 404,
+  //     msg: "User not found",
+  //   });
+  // }
 
-  if (await checkArticleExists(article_id)) {
-    return Promise.reject({
-      status: 404,
-      msg: "Articles not found",
+  // if (await checkArticleExists(article_id)) {
+  //   return Promise.reject({
+  //     status: 404,
+  //     msg: "Articles not found",
+  //   });
```

Comment:
Your commit shouldnt include commented out code. Please remove it from the commit